test/handlers_test.go

```diff
@@ -121,3 +123,34 @@ func TestHandlerTargetForwarding(t *testing.T) {
 		})
 	})
 }
+
+func TestHandlerSubrequestResolve(t *testing.T) {
+	t.Parallel()
+
+	runTest(t, func(ctx context.Context, gwc gwclient.Client) {
```

Comment:
Is it possible to add a test with multiple platforms?

Reply:
Done, but need #353  to make tests pass